test/test.js

```diff
@@ -119,6 +128,32 @@ describe('Reporting errors', function () {
     }
   });
 
+  it('should extract and send functionName in stack traces', function (done) {
+    var message = 'custom message';
+    // PhantomJS only attaches a stack to thrown errors
+    try {
+      throwError(message)
+    } catch(e) {
+      errorHandler.report(e, function() {
+        expectRequestWithMessage('throwError');
+        done();
+      });
+    }
+  });
+
+  it('should extract and send <anonymous> in stack traces', function (done) {
```

Comment:
Could you update the test name to 
"should set <anonymous> in stack traces when frame is anonymous"